awscli/customizations/wizard/wizards/dynamodb/new-table.yml

```diff
@@ -152,7 +258,24 @@ execute:
             Enabled: true
             SSEType: KMS
             KMSMasterKeyId: "{kms_key_id}"
-  apicalls:
     - type: apicall
       operation: dynamodb.CreateTable
       params: "{create_table_params}"
+__OUTPUT__:
+  value: |
+    Wizard successfully created DynamoDB Table:
+      Table name: {table_name}
+      Primary partition key: {primary_key_name} ({primary_key_type})
+      {% if wants_sort_key == yes %}
+      Primary sort key: {sort_key_name} ({sort_key_type})
+      {% endif %}
```

Comment:
It would be nice to add a newline after this in order to break up the summary section from the sample CLI commands and CloudFormation template sections.